styles.css

```diff
@@ -609,7 +605,7 @@ input::-webkit-input-placeholder {
   line-height: 36px;
   color: #fff;
   display: block;
-  margin: 0 0 0 47px;
+  text-align: end;
 }
```

Comment:
link shouldn't be clickable outside the text
![1](https://user-images.githubusercontent.com/6584990/184546911-9012d8e5-b3c9-4380-bde7-ff795bbb335f.png)